bears/ruby/RubySecurityBear.py

```diff
@@ -0,0 +1,24 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+
+
+@linter(executable='brakeman',
+        output_format='regex',
+        output_regex=r'L(?P<line>\d+)C(?P<column>\d+): (?P<message>.*)')
+class RubySecurityBear:
+    """
+    Checks the Security issues of Ruby Applications.
+    It uses `brakeman -p filename`.
+    """
+
+    LANGUAGES = {'Ruby'}
+    REQUIREMENTS = {GemRequirement('brakeman', '4.1.1')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Security'}
+    SEE_MORE = 'https://brakemanscanner.org/'
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return '-p', filename,
```

Comment:
why didnt you wrap `p` in the create arguments? It sud be right @Makman2 ?

Reply:
oh sorry @newbazz. It was re-pushed because of some testing. BTW now I'm doing `return filename` only. 😃